src/utility_belt/files.clj

```diff
@@ -0,0 +1,39 @@
+(ns utility-belt.files
+  (:require [clojure.edn :as edn]
+            [clojure.java.io :as io]
+            [cheshire.core :as json]
+            [clojure.data.csv :as csv]))
+
+
+(defn load-edn
+  "Given a resource name or file path, reads the EDN file and returns a Clojure data structure."
+  [resource-name-or-path]
+  (let [resource (or (io/resource resource-name-or-path)
+                     resource-name-or-path)]
+    (with-open [reader (java.io.PushbackReader. (io/reader resource))]
```

Comment:
I don't think we need the whole push back reader machinery - `(-> name io/resource slurp edn/read-string)` will be sufficient